### PR TITLE
fix: exclude popularity sort as eligible for the experiment

### PR DESCRIPTION
**Pull request by mcstover**

### CODE REVIEW

File: src/pages/Lend/Filter/LendFilterPage.vue [hunk 1]
```diff
@@ -143,7 +143,7 @@ function isFLSSEligible(route = {}) {
 	// check route for eligibility
 	const eligibleQueryParams = ['page', 'sortBy', 'gender', 'registration'];
 	const queryParamKeys = Object.keys(route?.query);
-	const allowedSorts = ['expiringSoon', 'loanAmountDesc', 'loanAmount', 'popularity'];
+	const allowedSorts = ['expiringSoon', 'loanAmountDesc', 'loanAmount']; // 'popularity'
```
Comment (dyersituations) [inline src/pages/Lend/Filter/LendFilterPage.vue:146]: Do we also need to remove "Recommended" from the available alpha sorts in the UI?

**Review by dyersituations — APPROVED** [review]

File: src/pages/Lend/Filter/LendFilterPage.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/Lend/Filter/LendFilterPage.vue:146]: Recommended is the default FLSS sort. I never actually went through mapping popularity to recommended which was good b/c we determined that wouldn't be what we'll do. Popularity is distinct. We do have a switch to flip to "popularity" when the query type isn't FLSS but I don't see a way that would flip at this time.

Comment (mcstover) [reply src/pages/Lend/Filter/LendFilterPage.vue:146]: Also, "Recommended" is the Label mapped from "personalized" for FLSS.

Comment (dyersituations) [reply src/pages/Lend/Filter/LendFilterPage.vue:146]: Oh right, recommended maps to personalized, not popularity.

**Pull request merged**